backend/routes.py

```diff
@@ -23,13 +25,21 @@ def chat_to_interface(chat: ct.Chat) -> dict:
         "isBookmarked": chat.bookmarked,
         "createdAt": chat.created.isoformat(),
         "updatedAt": chat.last_update.isoformat(),
-        "agentId": str(chat.agent_id)
+        "agentId": str(chat.agent_id),
     }
 
 
 CHAT_STORE_PROXY = copy.deepcopy(bm.MOCK_CHAT_STORE)
 
 
+@pike_router.get("/skills")
+def get_skills() -> list[dict]:
+    """
+    Get a list of all available skills.
+    """
+    return sk.Skill.get_all_skills()
```

Comment:
it is better to make `get_all_skills` an independent function, not a class method. It shouldn't depend on a state of an object.